src/prebid.js

```diff
@@ -442,6 +450,7 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id, options) {
 
           if (isRendererRequired(renderer)) {
             executeRenderer(renderer, bid);
+            emitAdRenderSucceeded({ doc, bid, id });
```

Comment:
Is it possible for the `executeRenderer` function to throw an error? If so, should this be in a `try/catch`?

Reply:
I'm not familiar with the renderer code, but logically, it should be able to fail one way or another. None of these functions down this chain return anything explicitly. I'll dig into this and see if i can come up with a testcase that can make executeRenderer explode.  Presumably if this can fail, then there is somewhere in here core should emitAdRenderFailed that it currently isn't.

Reply:
@msm0504, after looking at this a bit more, this section of code is already part of a big try/catch.  Most of this function is in it, and the catch already does emit the failure. So i think this is probably ok as is.  It doesn't look like there are tests for a renderer that actually throws something though, so i'm looking into adding some, but i'm not sure that should be part of this PR.